dataactcore/scripts/pull_fpds_data.py

```diff
@@ -1588,18 +1586,17 @@ def parse_fpds_file(f, sess, sub_tier_list, naics_dict, filename=None):
         "reasonnotcompeted", "numberofoffersreceived", "commercialitemacquisitionprocedures",
         "commercialitemtestprogram", "smallbusinesscompetitivenessdemonstrationprogram", "a76action",
         "competitiveprocedures", "solicitationprocedures", "typeofsetaside", "localareasetaside", "evaluatedpreference",
-        "fedbizopps", "research", "statutoryexceptiontofairopportunity", "organizationaltype", "numberofemployees",
-        "annualrevenue", "firm8aflag", "hubzoneflag", "sdbflag", "issbacertifiedsmalldisadvantagedbusiness",
-        "shelteredworkshopflag", "hbcuflag", "educationalinstitutionflag", "womenownedflag", "veteranownedflag",
-        "srdvobflag", "localgovernmentflag", "minorityinstitutionflag", "aiobflag", "stategovernmentflag",
-        "federalgovernmentflag", "minorityownedbusinessflag", "apaobflag", "tribalgovernmentflag", "baobflag",
-        "naobflag", "saaobflag", "nonprofitorganizationflag", "isothernotforprofitorganization",
-        "isforprofitorganization", "isfoundation", "haobflag", "ishispanicservicinginstitution",
-        "emergingsmallbusinessflag", "hospitalflag", "contractingofficerbusinesssizedetermination",
-        "is1862landgrantcollege", "is1890landgrantcollege", "is1994landgrantcollege", "isveterinarycollege",
-        "isveterinaryhospital", "isprivateuniversityorcollege", "isschoolofforestry",
-        "isstatecontrolledinstitutionofhigherlearning", "isserviceprovider", "receivescontracts", "receivesgrants",
-        "receivescontractsandgrants", "isairportauthority", "iscouncilofgovernments",
+        "fedbizopps", "research", "statutoryexceptiontofairopportunity", "organizationaltype", "firm8aflag",
```

Comment:
I don't think we should remove the columns from the historical load because if we're keeping the columns and ever have to reload this will remove the historical data and make keeping the columns moot.

Reply:
Added them back in 👍